README.adoc

```diff
@@ -267,45 +266,46 @@ The methods that you wrote in the [hotspot=Consumer file-0]`Consumer` class coul
 party service, you should separate your `GET`/`POST` requests from your data consumption.
 
 // =================================================================================================
-// Building and running the application
+// Running the application
 // =================================================================================================
 
-[role=command]
-include::{common-includes}/mvnbuild.adoc[]
+== Running the application
 
-When the server is running, you can find your service at http://localhost:9080/artists[http://localhost:9080/artists^]
+The Open Liberty server was started in development mode at the beginning of the guide and all the changes were automatically picked up.
 
-[role=command]
-include::{common-includes}/mvncompile.adoc[]
+You can find your service at http://localhost:9080/artists[http://localhost:9080/artists^].
```

Comment:
Change
"You can find your service at http://localhost:9080/artists[http://localhost:9080/artists^]."
To
"You can find your service at the http://localhost:9080/artists[http://localhost:9080/artists^] URL."